spec/kitchen/driver/ec2/image_selection_spec.rb

```diff
@@ -68,6 +68,19 @@ def new_instance(platform_name: "blarghle")
       { :name => "architecture", :values => ["x86_64"] },
     ],
 
+    "amazon2" => [
+      { :name => "owner-id", :values => %w{137112412989} },
+      { :name => "name", :values => %w{amzn2-ami-hvm-2.0.*} },
+    ],
+    "amazon2-2017" => [
```

Comment:
Do we really need to support the 2017 amazon linux. That was just the RC release where they used the old naming schema. They haven't made it entirely clear how they plan to name the new 2.x series.